### DISCUSSION TITLE
how to check long filename

### DISCUSSION THREAD

**Discussion by tkkcc** [nvim-neo-tree/neo-tree.nvim#1334] [Q&A] [answered]
![image](https://github.com/nvim-neo-tree/neo-tree.nvim/assets/17373509/a50b3fa6-fd04-4ac5-9808-c3f9836fc0a5)

The only way i know to distinguish above files, is to resize the panel.

I wonder if there is any better ways, for example showing filename in a `preview window` like diagnostics info.

BTW, in nvimtree, i used to scroll right using `l` to see the filenames.

**Comment by pysan3** [comment, marked as answer]
Press `e`. It's called _expand_width_ and you also have an option `config.window.auto_expand_width` to make it the default.

**Comment by cseickel** [comment]
You can also press `i` to show "information" which will be a popup window with details about the file.
